infrastructure/validators/rewards-payout.md

```diff
@@ -0,0 +1,225 @@
+
+??? function "Probability in Staking Rewards"
+
+    This should only serve as a high-level overview of the probabilistic nature for staking rewards.
+
+    Let:
+
+    - `pe` = para-validator era points,
+    - `ne` = non-para-validator era points,
+    - `EV` = expected value of staking rewards,
+
+    Then, `EV(pe)` has more influence on the `EV` than `EV(ne)`.
+
+    Since `EV(pe)` has a more weighted probability on the `EV`, the increase in variance against the `EV` becomes apparent between the different validator pools (aka. validators in the active set and the ones chosen to para-validate).
+
+    Also, let:
+
+    - `v` = the variance of staking rewards,
+    - `p` = number of para-validators,
+    - `w` = number validators in the active set,
+    - `e` = era,
+
+    Then, `v` &#8593; if `w` &#8593;, as this reduces `p` : `w`, with respect to `e`.
+
+    Increased `v` is expected, and initially keeping `p` &#8595; using the same para-validator set for all parachains ensures [availability](TODO: add path) and [voting](../learn/learn-polkadot-opengov.md). In addition, despite `v` &#8593; on an `e` to `e` basis, over time, the amount of rewards each validator receives will equal out based on the continuous selection of para-validators.
+
+    There are plans to scale the active para-validation set in the future.
+
+## Payout Scheme
+
+Validator rewards are distributed equally among all validators in the active set, regardless of the total stake behind each validator. However, individual payouts may differ based on the number of era points a validator has earned. Although factors like network connectivity can affect era points, well-performing validators should accumulate similar totals over time.
+
+Validators can also receive tips from users, which incentivize them to include certain transactions in their blocks. Validators retain 100% of these tips.
+
+Rewards are paid out in the network's native token (KSM for Kusama and DOT for Polkadot). 
+
+The following example illustrates a four member validator set with their names, amount they have staked, and how payout of rewards is divided. This scenario assumes all validators earned the same amount of era points and no one received tips: 
+
+``` mermaid
+%%Payout, 4 val set, A-D are validators/stakes, E is payout%%
+
+block-beta
+    columns 1
+  block
+    A["Alice (18 DOT)"]
+    B["Bob (9 DOT)"]
+    C["Carol (8 DOT)"]
+    D["Dave (7 DOT)"]
+  end
+    space
+    E["Payout (8 DOT total)"]:1
+    E --"2 DOT"--> A
+    E --"2 DOT"--> B
+    E --"2 DOT"--> C
+    E --"2 DOT"--> D 
+```
+
+Note that this is different than most other Proof-of-Stake systems. As long as a validator is in the validator set, it will receive the same block reward as every other validator. Validator Alice, who had 18 tokens staked, received the same 2 token reward in this era as Dave who had only 7 tokens staked.
```

Comment:
```suggestion
Note that this is different than most other Proof-of-Stake systems. As long as a validator is in the validator set, it will receive the same block reward as every other validator. Validator Alice, who had 18 DOT staked, received the same 2 DOT reward in this era as Dave, who had only 7 DOT staked.
```

Reply:
Just to match the diagram 🤷‍♀️